runtime/doc/helphelp.txt

```diff
@@ -368,15 +368,5 @@ highlighting.  So do these:
 
 You can find the details in $VIMRUNTIME/syntax/help.vim
 
-							*inclusion*
-Some people make a big deal about using "his" when referring to the user,
-thinking it means we assume the user is male.  That is of course not the case,
-it's just a habit of writing help text, which quite often is many years old.
-Also, a lot of the text is written by contributors for who English is not
-their first language.  We do not make any assumptions about the gender of the
-user, no matter how the text is phrased.  And we do not want to waste time on
-this discussion.  The goal is that the reader understands how Vim works, the
-exact wording is secondary.
```

Comment:
Why is this part being removed? It solves the very problem this PR is intending to solve, so it feels like this is being removed in order to introduce a problem, to *then* justify this PR. Specifically, the part "And we do not want to waste time on this discussion", which would prohibit this PR to begin with.

This PR reeks of virtue-signalling.

Reply:
@Tyil: I strongly disagree with your statement. I am admittedly surprised that this pull request is seen as an act of virtue signalling when it simply attempts to remove harmful language from the documentation in order to show consideration for underrepresented groups in our industry.

From my previous comments, I think it should be clear why this text does not solve the issue but let me dissect it to further stress my argument.
In the first sentence, two issues are apparent. First, it neglects how using "his" is in fact an issue of great importance to a lot of people, and, secondly, it notes that the use of "his" does not imply that the user is male as some people may think. By using "his", I think it is only natural to make this assumption. For anybody who identifies as non-male, I imagine that it may seem incredibly counterintuitive why the documentation is so hesitant to adopt singular "they" instead of gendered pronouns if this is the case. Why not simply apply a pronoun that carries no assumptions about the gender if it is so important to keep stating that there are no assumptions about the gender? Using singular "they" removes the need to do so. Then it goes on to explain that the use of "his" is just the result of a habit that has developed throughout the years. In the past, we have used hurtful words to describe various groups of people but we have shown time and time again that we can change habits for the better. I do not see why the Vim documentation should not welcome change of this form too. Finally, the text once again mentions that it "do[es] not want to waste time on this discussion" which returns to the theme of neglecting how this issue may be hugely important to a lot of people.

Reply:
To support @majjoha opinion here, there are two things I would like to add about this removed part.

- First, I came accross this text long after I had started reading the docs. I understand that this is not the way the docs are intended to be read, and that maybe I should have read it first. But I bet many people do the same. Should they feel offended by the rest of the doc, then it would be too late and it's not just "their fault". Especially now that this PR exists and that the burden of editing the whole doc has been taken in charge by @majjoha.

- Second, even though I don't personally feel offended by this removed part. Here is how I understand/rephrase it in my head:

> We are aware that the exact wording of the doc is likely to offend people. However, we think it is secondary matter so we did not intend to fix it.

.. which I feel offending.

Reply:
> Why is this part being removed? It solves the very problem this PR is intending to solve.

A declaration that no harm is meant is not sufficient to do no harm. It is in a place that few users of the documentation will ever see. And "you might find this offensive but I assure it is not offensive" doesn't actually help a person who feels excluded.